bin/vault_utils/vault_interface.py

```diff
@@ -38,18 +38,20 @@ def _authenticate_approle(self):
             "auth/{0}/login".format(self._approle_path)
         )
 
-        r = requests.post(auth_url, data=auth_data)
+        headers = {}
+        self._add_namespace(headers)
+        r = requests.post(auth_url, data=auth_data, headers=headers)
+        r.raise_for_status()
```

Comment:
This should have already been in place, but wasn't.  Previous testing was done using a root namespace for authentication, but a child namespace when fetching credentials.  This needed to be added to enable child namespace-mounted authentication paths for the initial authentication.